### PR TITLE
Rate limit notifications

### PR DESCRIPTION
**Pull request by jcontonio**
* Adds rate limit notifications
* Reduces the amount of notifications to 1 per error code
* Fixes a few rendering issues in components that weren't checking for things first

### CODE REVIEW

File: web/src/client/js/components/Notifications/NotificationList.js [hunk 1]
```diff
@@ -4,15 +4,21 @@ import PropTypes from 'prop-types'
 import NotificationComponent from './NotificationComponent'
 
 const NotificationList = ({ dismissHandler, notifications }) => {
-  function renderNotices() {
-    return notifications.map((notification) => {
-      return <NotificationComponent dismissHandler={dismissHandler} key={notification.id} id={notification.id} notification={notification.notification} />
-    })
-  }
-  if (notifications.length > 0) {
+  console.log(notifications)
```
Comment (dheniges) [inline web/src/client/js/components/Notifications/NotificationList.js]: 👀

File: web/src/client/js/components/Notifications/NotificationList.js [hunk 2]
```diff
@@ -4,15 +4,21 @@ import PropTypes from 'prop-types'
...
+  const codeArray = []
+  const trimmedNotificationsList = notifications.filter((note) => {
+    if (!codeArray.includes(note.notification.code)) {
+      codeArray.push(note.notification.code)
+      return note
+    }
+  })
+  console.log(trimmedNotificationsList)
```
Comment (dheniges) [inline web/src/client/js/components/Notifications/NotificationList.js]: :eyes:

**Review by dheniges — CHANGES_REQUESTED** [review]
Everything looks good except those console logs 👀

**Comment by jcontonio** [comment]
Whoooooooops

**Review by dheniges — APPROVED** [review]
👍

**Pull request merged**